CHANGELOG.md

```diff
@@ -41,10 +41,33 @@
 
 
 ### IMPROVEMENTS:
-* core: Bump github.com/hashicorp/hcp-sdk-go from 0.90.0 to 0.92.0.
+* core: Bump github.com/hashicorp/hcp-sdk-go from 0.90.0 to 0.96.0.
      [GH-12935](https://github.com/hashicorp/packer/pull/12935)
+     [GH-12942](https://github.com/hashicorp/packer/pull/12942)
+     [GH-12960](https://github.com/hashicorp/packer/pull/12960)
+     [GH-12979](https://github.com/hashicorp/packer/pull/12979)
 * core: Bump github.com/hashicorp/packer-plugin-sdk from 0.5.2 to 0.5.3
      [GH-12932](https://github.com/hashicorp/packer/pull/12932)
+* .github: updated actions in preparation for Node 16 deprecation
```

Comment:
We generally don't include these in the CHANGELOG because they can add a lot of noise. I generally treat them as tech-debt.

Reply:
Yeah sounds good to me, I wasn't sure if we should include them, but leaving them out is probably for the best as users don't really care about maintenance of the repo (nor should they)